src/run_ci.py

```diff
@@ -9,18 +9,31 @@
 import logging
 import sys
 
+import yaml
+
 from ci_workflow.ci_args import CiArgs
 from ci_workflow.ci_check_lists import CiCheckLists
 from ci_workflow.ci_target import CiTarget
 from manifests.input_manifest import InputManifest
+from manifests.test_manifest import TestManifest
 from system import console
 from system.temporary_directory import TemporaryDirectory
 
 
 def main():
     args = CiArgs()
     console.configure(level=args.logging_level)
-    manifest = InputManifest.from_file(args.manifest)
+
+    manifest_type = __get_manifest_type(args.manifest)
+
+    if manifest_type == "Test":
+        TestManifest.from_path(args.manifest.name)
+
+        logging.info("TestManifest schema validation succeeded")
+        logging.info("Done.")
+        return
+
+    manifest = InputManifest.from_path(args.manifest.name)
```

Comment:
Shouldn't CI call have always expected a TestManifest type?

Reply:
The existing manifests https://github.com/opensearch-project/opensearch-build/tree/main/manifests/1.3.0 are `InputManifest` which is different from the `TestManifest` newly added in this PR.